### ISSUE TITLE
Connector Workflow is not created on Connector Wizard close/create

### ISSUE THREAD

**Issue by robplank** [robplank/azure-automation-webhook-scsm-connector#55]
when the New Connector wizard is completed the Connector workflow doesn't get created, not seeing the issue when attaching a debugger.

**Comment by JakobGSvendsen** [comment]
Hmm I am having trouble even connecting the debugger and using a breakpoint. I just attach to the console service, is that correct? how did you make it actually stop on breakpoints?

**Comment by robplank** [comment]
yes set your break points in VS then attach the debugger, open the wizard to create a new connector select the Azure Automation connector and you should hit your break point. try pulling down the issue_7_Activity branch the workflow creation is completely rewritten based on what I learned from the Exchange connector. the error should be on line 532 of AzureAutomation.cs when attempting to save the type projection.

**Comment by JakobGSvendsen** [comment]
ok got it now, but I need to find some references again as they are missing in the reference folder beofre I can test... :)

**Comment by JakobGSvendsen** [comment]
line 532 is     public static class StringCipher ?

**Comment by JakobGSvendsen** [comment]
tried to break here:

![2016-10-15_17-39-07](https://cloud.githubusercontent.com/assets/3460807/19414341/60e5036a-92fe-11e6-9fce-6aa67990f12a.png)

and attaches to the consiole process:

![2016-10-15_17-39-30](https://cloud.githubusercontent.com/assets/3460807/19414342/6b8db8d4-92fe-11e6-935e-f9595c128be1.png)

but it never breaks for some reason! can't figure out what i have done wrong, I have attached by remote lots of times in SCSM projects without issues, it seems like it does not execute the code.

I have also tried to insert a messagebox:show but no box shows ( but this can be because of where the code runs in SCSM)

**Comment by robplank** [comment]
Did you copy the dlls from the VS build to the SM install folder. see issue #22
